content/maxwell3_fdem/emi_tutorial/mutualinductance.rst

```diff
@@ -0,0 +1,136 @@
+.. _mutualinductance:
+
+Mutual inductance
+=================
+
+What is mutual inductance?
+--------------------------
+
+When an EMF (:math:`\mathcal{E}`) is produced in a loop because of the change in current in a coupled loop, the effect is called mutual inductance.
```

Comment:
An overview / purpose statement here as well would be helpful
"In this page we..."